ga4gh/client.py

```diff
@@ -38,10 +36,13 @@ def __init__(self, urlPrefix, debugLevel, workarounds, key):
         else:
             logLevel = logging.DEBUG
         self._logger.setLevel(logLevel)
-        handler = logging.StreamHandler()
-        formatter = logging.Formatter('%(name)s: %(message)s')
-        handler.setFormatter(formatter)
-        self._logger.addHandler(handler)
+
+        requestsLog = logging.getLogger("requests.packages.urllib3")
```

Comment:
Dunno - seems like they were getting sent to the requests logger as well as the one we'd instantiated.